pkg/graveler/sstable/helpers.go

```diff
@@ -0,0 +1,16 @@
+package sstable
+
+import "github.com/cockroachdb/pebble"
+
+func retrieveValue(lazyValue pebble.LazyValue) ([]byte, error) {
+	val, owned, err := lazyValue.Value(nil)
+	if err != nil {
+		return []byte{}, err
```

Comment:
```suggestion
		return nil, err
```
Generally we should avoid using an empty slice as distinct from `nil`.

Reply:
done